partner_second_lastname/models/res_partner.py

```diff
@@ -85,8 +85,8 @@ def _get_inverse_name(self, name, is_company=False):
                     parts = result['firstname'].split(" ", 1)
                 while len(parts) < 2:
                     parts.append(False)
-                result['lastname2'] = parts[0]
-                result['firstname'] = parts[1]
+                result['firstname'] = parts[0]
```

Comment:
I think this is wrong.

Check line 83: `if order == 'last_first':`. So, I understand lastname goes first with this setting.

Reply:
@Yajo 

could you review again please